.github/workflows/lint-typescript.yml

```diff
@@ -4,7 +4,9 @@
 
 name: Lint TypeScript
 
-on: pull_request
+# FIXME: temporary disable typecheck, until we can perform typecheck without checking Talk
```

Comment:
There's one more FIXME on line 20

P.S. we can't just ignore spreed directory, right?